pkg/internal/checkup/checkup_test.go

```diff
@@ -621,6 +623,37 @@ func (cs *clientStub) GetDataSource(ctx context.Context, namespace, name string)
 	return das, nil
 }
 
+func (cs *clientStub) GetClusterVersion(ctx context.Context, name string) (*configv1.ClusterVersion, error) {
```

Comment:
what about a UT to make sure you get the last updated version? and maybe also negative testing.. although it is pretty straight foreword function..

Reply:
Will add the positive one :)